components/Metrics.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+  <div>
+    <div
+      v-for="(metric, index) in metrics"
+      :key="metric.metric"
+      class="flex flex-row justify-between mb-8"
+    >
+      <div class="bg-purple-700 text-white w-32 mr-8 p-3 rounded-xl">
+        {{ metric.metric }}
+      </div>
+      <div class="flex" style="width: 500px">
+        <p class="mr-1 self-center">
+          {{ metric.measure.min }}
+        </p>
+        <div
+          class="flex justify-center place-content-evenly font-bold items-center border border-black"
+          :style="metricWidth"
+        >
+          <div
+            v-for="i in numberOfBoxes(index)"
+            :key="`box_${metric.metric}_${i}`"
+            :class="
+              boxStyleClasses(i, numberOfBoxes(index), metric.measure.value)
+            "
+            :style="boxWidth(metric.measure)"
+          ></div>
+        </div>
+        <p class="ml-1 self-center">
+          {{ metric.measure.max }}
+        </p>
+      </div>
```

Comment:
### Sintaxe carregada

Pensei aqui que talvez ficasse melhor se você trocasse o `v-for` para usar o object destruction, algo assim:
```vue
v-for="({ metric, measure }, index) in metrics"
```

Dessa forma, as coisas ficariam mais enxutas. Ex: na linha 29 ficaria
```diff
+ {{ measure.max }}
- {{ metric.measure.max }}
```

---

### Componentização

Fiquei pensando aqui que poderia ser uma boa ter um componente dedicado para ser uma régua, que você apenas o utilizaria para ficar ao lado do nome da métrica, saca? Vê o que você acha, porque pode ser um comentário pré-maturo da minha parte.

Reply:
Eu experimentei fazer isso: quebrei esse componente grande em dois novos componentes, um com a lista de métricas e outro para a régua. Gostei do resultado porque o componente de régua era a parte que tinha mais lógica, então ficou abstraído da disposição da lista em si. Dei commit e já vou dar merge também.